test/test_flush_refresh.sh.in

```diff
@@ -62,6 +62,16 @@ srcdir=@srcdir@
 utils_testdir=@abs_top_builddir@/@H5_UTILS_TEST_BUILDDIR@
 testdir=@abs_top_builddir@/@H5_TEST_BUILDDIR@
 
+# If the bindir directory is not set just use current (.).
+if test -z "$bindir"; then
+   bindir=.
+fi
+
+# If the testdir directory is not set just use current (.).
+if test -z "$testdir"; then
+   testdir=.
+fi
+
```

Comment:
What does this have to do with this PR?

Reply:
Looks like old code - might be some merges are needed from latest.

Reply:
I don't know - these changes have been here since I took over the branch. I can try removing them

Reply:
All commits should have been merged as of a week or so ago